### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Review by Debarshi95 — COMMENTED** [review]
All good. One suggestion, try to use semantic tags like article section etc along with div tags.

**Review by Debarshi95 — APPROVED** [review]

**Comment by gadgetfather** [comment]
The "shop now" button is overlapping other content.  Also, you can reduce the height of hero since its covering too much space and the user will not be able to see what he/she has to do.
### Oversized hero image
image
----------------------------------------------------------------------------------------------------------------------------------
### Shop now button overlapping
image

File: common.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  .page-wrapper {
+    height: max-content;
+    margin: 0;
+    z-index: 10;
+  }
+  *
+  {
+      padding:0;
+      box-sizing: border-box;
+  }
```
Comment (gadgetfather) [inline common.css]: You can write this at the top of file, makes it easier to read

File: style.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    border-radius:10px;
+    color: white;
+    background-color: black;
+    text-decoration: none;
+}
+.overlay-text
+{
+  position: absolute;
+  top: 110%;
+  left: 42%;
```
Comment (gadgetfather) [inline style.css]: Here you can give values in rem

**Review by gadgetfather — APPROVED** [review]

File: common.css (hunk 1 above)
Comment (ShrutiLakhani) [reply common.css]: Made changes

**Pull request merged**